### ISSUE TITLE
Gemini consult

### ISSUE THREAD

**Issue by wanshangfeng** [Yunlongs/Yunlongs#91]
Could you share the Gemini source code? It will be very helpful for me. 
Regards

**Comment by Yunlongs** [comment]
@wanshangfeng yeah, the source code of Gemini is already shared, please see: https://github.com/Yunlongs/Gemini
